compose.yml

```diff
@@ -37,7 +37,7 @@ services:
             - MW_DB_INSTALLDB_USER=root
             - MW_DB_INSTALLDB_PASS
             - MW_DB_USER=root
-            - MW_DB_PASS=$MW_DB_INSTALLDB_PASS
+            - MW_DB_PASS=/run/secrets/db_root_password
```

Comment:
This won't work by itself, please see how it's done here https://github.com/WikiWorks/docker-wikicity.com/blob/main/config/LocalSettings.php#L96